etc/org.opencastproject.serviceregistry.impl.ServiceRegistryJpaImpl.cfg

```diff
@@ -7,7 +7,7 @@ max.attempts=1
 # a mimimum value of 1s is enforced due to performance reasons. Set to 0 to disable dispatching from this service
 # registry.
 #Service registry dispatching should be set to 0 on everything but admin or allinone
-#dispatchinterval=5000
+#dispatch.interval=5
```

Comment:
Needs the reference above this changed: "# The interval in milliseconds"

Reply:
Also I wonder if there are actually any performance implications that need this to be so slow, or if we could change the default to 1s.

Reply:
A sufficiently heavily loaded system can take longer than 1 second to do a full round of dispatching :(

Reply:
Yes but that doesn't matter because the setting is the interval between dispatch runs:

https://github.com/opencast/opencast/blob/develop/modules/serviceregistry/src/main/java/org/opencastproject/serviceregistry/impl/ServiceRegistryJpaImpl.java#L836

https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ScheduledExecutorService.html#scheduleWithFixedDelay(java.lang.Runnable,%20long,%20long,%20java.util.concurrent.TimeUnit)

We don't need to resolve that question for this PR but I don't think it needs to be that slow.

Reply:
Changed to seconds